R/calcFEdemand.R

```diff
@@ -606,6 +626,11 @@ calcFEdemand <- function(subtype = "FE") {
                                              dim = "item", na.rm = TRUE)
     }
 
+    # remove missing Navigate scenarios
```

Comment:
will they be added later?

Reply:
No, the code requires that all SSP* scenarios have all RCP scenarios. But I only provide the new Navigate scenarios without RCP variants. So I [fill with NAs](https://github.com/pik-piam/mrremind/pull/305/files#diff-e95c22d751b629b87e26a14288055f36382b824991dc3eeec561ae21aa254daaR366-R368) before and finally remove them here.